### PR TITLE
Feature/mongodb atlas

### PR DESCRIPTION
**Pull request by Hwid2121**

### CODE REVIEW

**Comment by Hwid2121** [comment]
@micheledallerive  please help me to merge this 👉👈

File: models/index.js [hunk 1]
```diff
@@ -10,15 +10,17 @@ const client = new MongoClient(mongodb_uri, {
 });
 */
 
+const connectionParams = {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+};
```
Comment (micheledallerive) [inline models/index.js:16]: Put inside the setup function.

File: models/index.js [hunk 2]
```diff
@@ -10,15 +10,17 @@ const client = new MongoClient(mongodb_uri, {
 });
 */
 
+const connectionParams = {
```
Comment (micheledallerive) [inline models/index.js:13]: Change name, I don't like it.

File: models/index.js [hunk 3]
```diff
@@ -10,15 +10,17 @@ const client = new MongoClient(mongodb_uri, {
...
   try {
-    await mongoose.connect(process.env.MONGODB_URI, {
-      useNewUrlParser: true,
-      useUnifiedTopology: true,
-    });
+    await mongoose.connect(process.env.MONGODB_URI, connectionParams);
+    console.log('✅Connected to database');
   } catch (err) {
-    console.error(err);
+    console.error(`Error connecting to the database. \n${err}`);
```
Comment (micheledallerive) [inline models/index.js:22]: Don't do \n
```suggestion
    console.error(`Error connecting to the database: ${err}`);
```

File: models/index.js [hunk 4]
```diff
@@ -10,15 +10,17 @@ const client = new MongoClient(mongodb_uri, {
...
   }
 }
-
+// process.env.MONGODB_URI
```
Comment (micheledallerive) [inline models/index.js:25]: ??

**Review by micheledallerive — CHANGES_REQUESTED** [review]

**Review by DonnaDona — APPROVED** [review]
i guess

**Review by JekxDevil — APPROVED** [review]

**Review by micheledallerive — APPROVED** [review]

**Pull request merged**